src/algorithms/rectangularcluster.jl

```diff
@@ -6,14 +6,20 @@ rectangle/patch is between `minimum` and `maximum` (the two can be equal for a
 fixed size rectangle).
 """
 struct RectangularCluster <: NeutralLandscapeMaker
-    minimum::Integer
-    maximum::Integer
-    function RectangularCluster(x::T, y::T) where {T <: Integer}
+    minimum::Int
```

Comment:
I guess in principle we could have unsigned integers here, which would now fail, but to be fair they are relatively rare in the ecosystem